### ISSUE TITLE
Unknown arguments to `#[instrument]` are ignored

### ISSUE THREAD

**Issue by mcarton** [tokio-rs/tracing#772]
### Version

```
└── tracing v0.1.15
    ├── tracing-attributes v0.1.8
    └── tracing-core v0.1.10
```

### Crates

- `tracing-attributes`

### Description

I made the mistake to use
```rust
#[instrument(foo="bar", id=1, show=true)]
fn foo() {}
```

instead of

```rust
#[instrument(fields(foo="bar", id=1, show=true))]
//           ^^^^^^^                          ^
fn foo() {}
```

This is silently accepted by the attribute, but does nothing. I'd expect an error that `foo`, `id` and `show` don't exist, with a possible suggestion to use `fields`.

**Comment by hawkw** [comment]
I believe this was fixed in PR #672 --- but it hasn't been released yet. We should get on that!

**Comment by hawkw** [comment]
I'm closing this as it's a duplicate of #727. We'll try to publish the fix for it soon.
